tests/test_data_store_merge_platforms.py

```diff
@@ -0,0 +1,680 @@
+                sensor_2,
+                self.current_time,
+                parser_name=self.parser_name,
+            )
+            self.file.create_state(
+                self.store,
+                platform_2,
+                sensor_2,
+                self.current_time,
+                parser_name=self.parser_name,
+            )
+            self.file.create_contact(
+                self.store,
+                platform_2,
+                sensor_2,
+                self.current_time,
+                parser_name=self.parser_name,
+            )
+            self.file.create_contact(
+                self.store,
+                platform_2,
+                sensor_2,
+                self.current_time,
+                parser_name=self.parser_name,
+            )
+            if self.file.validate():
+                self.file.commit(self.store, self.change_id)
+
+            # Assert that target platform doesn't have any states or contacts
+            sensors_before_merge = (
+                self.store.session.query(Sensor).filter(Sensor.host == platform.platform_id).all()
+            )
+            states_before_merge = (
+                self.store.session.query(State).filter(State.sensor_id == sensor).all()
+            )
+            contacts_before_merge = (
+                self.store.session.query(State).filter(State.sensor_id == sensor).all()
+            )
+            assert len(sensors_before_merge) == 1
+            assert len(states_before_merge) == 0
+            assert len(contacts_before_merge) == 0
+
+            self.store.merge_platforms([platform_2.platform_id], platform.platform_id)
+
+            # There should be still one sensor, but that sensor should have two States and two Contacts
+            sensors_after_merge = (
+                self.store.session.query(Sensor).filter(Sensor.host == platform.platform_id).all()
+            )
+            states_after_merge = (
+                self.store.session.query(State).filter(State.sensor_id == sensor).all()
+            )
+            contacts_after_merge = (
+                self.store.session.query(State).filter(State.sensor_id == sensor).all()
+            )
+            assert len(sensors_after_merge) == 1
+            assert len(states_after_merge) == 2
+            assert len(contacts_after_merge) == 2
+
+            # Assert that sensor_2 doesn't have any measurements because they are moved
+            states_after_merge = (
+                self.store.session.query(State).filter(State.sensor_id == sensor_2.sensor_id).all()
+            )
+            contacts_after_merge = (
+                self.store.session.query(State).filter(State.sensor_id == sensor_2.sensor_id).all()
+            )
+            assert len(states_after_merge) == 0
+            assert len(contacts_after_merge) == 0
+
+    def test_merge_platforms_with_different_sensor_names(self):
+        """Create two platforms, each platform will have a sensor named TEST-SENSOR.
```

Comment:
I think this docstring is out of date with the test - this test has different sensor names

Reply:
Thanks for pointing out. I updated out of date docstrings and comments: https://github.com/debrief/pepys-import/pull/740/commits/94ce9d8240683472553fd09c225987e45e4f70b3